pallets/utility/src/lib.rs

```diff
@@ -292,7 +298,8 @@ decl_module! {
             signature: <T as IdentityTrait>::OffChainSignature,
             call: UniqueCall<<T as Trait>::Call>
         ) -> DispatchResultWithPostInfo {
-            let _ = ensure_signed(origin)?;
+            let sender = ensure_signed(origin)?;
+            CallPermissions::<T>::ensure_call_permissions(&sender)?;
 
```

Comment:
The call name, and module name in  `pallets/permissions` should be updated before the tx is relayed.